src/app/fillResultsModal.js

```diff
@@ -0,0 +1,37 @@
+export const fillResultsModal = (gameHandlerResults) => {
+
+    const resultsDescription = document.querySelector('#resultsDescription');
+    resultsDescription.textContent = `Congratulations ${gameHandlerResults.name}! During ${gameHandlerResults.time} seconds you answered ${gameHandlerResults.answers.length} questions and scored ${gameHandlerResults.score} points!`
+
+    const resultsTable = document.querySelector('#tableWithResults table');
+
+    for (let questionItem of gameHandlerResults.answers) {
+        
+        const tableCell = document.createElement('tr');
+        tableCell.className = 'tableWithResultsQA';
+        if ((questionItem.question).substring(0, 3) !== 'http') {
```

Comment:
oh, it's cute but implementation shouldn't base on question starting with 'http' :P It doesn't tell much to other people. It would be better to pass "mode" to fillResultsModal function (or maybe even better - make it a property of gameHandlerResults) and decide based on mode.